bucket/manifold-viewer-9-official.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
```

Comment:
This is nonnse description of application. Does not explain what it is for

Reply:
it is tautology, true. Will fix